test/Microsoft.TemplateEngine.Utils.UnitTests/WellKnownSearchFiltersTests.cs

```diff
@@ -19,7 +19,7 @@ public class WellKnownSearchFiltersTests
         public void TagFilterTests_TemplateWithTags(string templateTags, string testTag, MatchKind? kind)
         {
             const string separator = "||";
-            string[] templateTagsArray = templateTags.Split(separator);
+            var templateTagsArray = templateTags.Split(new[] { separator }, System.StringSplitOptions.None);
```

Comment:
Is there a behaviour change here or is this just cosmetic?

Reply:
needed .NET full support - seems doesn't have previous overload